var/spack/repos/builtin/packages/tbl2asn/package.py

```diff
@@ -14,12 +14,12 @@ class Tbl2asn(Package):
 
     homepage = "https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/"
 
-    version('2020-03-01', sha256='7cc1119d3cfcbbffdbd4ecf33cef8bbdd44fc5625c72976bee08b1157625377e')
+    version('2020-03-01', sha256='b7edf378c971e6d7c48bf4d391d43a6d79f0e26dfa57156b4f87456af83e1fee')
```

Comment:
It looks like the old file was deleted and this package is not under version control of any sort. Can you add
```
url = "https://ftp.ncbi.nih.gov/toolbox/ncbi_tools/converters/by_program/tbl2asn/linux64.tbl2asn.gz"
```
under the `homepage` above, add a `url="https://ftp.ncbi.nih.gov/toolbox/ncbi_tools/converters/by_program/tbl2asn/linux.tbl2asn.gz"` to the existing 2020-03-01 version, and add a new `2021-10-31` version with the new hash?

Reply:
tbl2asn only runs when it is within 1 year of its release date, meaning 2020-03-01 would not even work. I don't see the point of adding it.

Reply:
[tbl2asn] This copy of tbl2asn is more than a year old. Please download the current version.

Reply:
Holy shit... @adamjstewart here's an unusual version use case for you.

Reply:
It's unfortunate that tbl2asn doesn't use release versions. This makes science impossible to reproduce. I would still keep the checksum for the old version since Spack keeps each release in its downloads cache.